1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,61 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+const div = document.createElement("div");
+const pQuote = document.createElement("p");
+const pAuthor = document.createElement("p");
+const button = document.createElement("button");
+const labelToggle = document.createElement("label");
+const spanStatus = document.createElement("span");
+const check = document.createElement("input");
+const spanSlider = document.createElement("span");
+
+check.setAttribute("type", "checkbox");
+
+document.body.appendChild(div);
+div.appendChild(labelToggle);
+labelToggle.appendChild(spanStatus);
+labelToggle.appendChild(check);
+labelToggle.appendChild(spanSlider);
+div.appendChild(pQuote);
+div.appendChild(pAuthor);
+div.appendChild(button);
```

Comment:
see what I mentioned before how you're already grouping these into groups of functionality, you can split this into a seperate function with a nice name which explains what this grouping does